services/apps/src/apps/persistence/app_metadata.clj

```diff
@@ -710,3 +719,28 @@
                            {:u.username username})
                        {:aca.app_id (uuidify app-id)
                         :l.id       [not= (user-favorite-subselect :w.root_category_id faves-idx)]})))))
+
+(defn- list-duplicate-apps*
+  [app-name app-id category-id-set]
+  (select [:apps :a]
+          (fields :a.id :a.name :a.description)
+          (join [:app_category_app :aca] {:a.id :aca.app_id})
+          (where {(raw "trim(both from a.name)") (string/trim app-name)
```

Comment:
The default behavior of `trim` is to trim both sides with whitespace, so this can just be `(sqlfn trim :a.name)`. For extra equivalentness you could have the DB do the trimming of both sides of the equality but I'm not sure that's worth it.